bramkouwen/Kouwen_A_Rabari_A_AutomotiveMusicMixer
# Updating wave color

## What does the PR do?
* It changes the color of the sound waves


## How do you check this PR?
* Pull down the branch
*  See changed stroke color

## Screenshots 
None.
